aioshelly/rpc_device/device.py

```diff
@@ -538,3 +544,28 @@ def _parse_dynamic_components(self, components: dict[str, Any]) -> None:
                 for item in self._dynamic_components
             }
         )
+
+    async def _retrieve_blutrv_components(self, components: dict[str, Any]) -> None:
+        """Retrieve BLU TRV components."""
+        if self.model != MODEL_BLU_GATEWAY_GEN3:
+            return
+
+        if not self._config or not self._status:
+            raise ConnectionError
```

Comment:
```suggestion
            raise NotInitialized
```
We don't have any `ConnectionError` handling in `aioshelly`